spec/e2e/support/target-blank.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * @typedef SimplifiedLink
+ *
+ * @prop {string=} innerText
+ * @prop {string=} href
+ * @prop {string=} className
+ * @prop {string=} target
+ */
+
+/** @type {import('puppeteer').Page} */
+/** @return {Promise<SimplifiedLink[]>} */
+async function getLinks(page) {
+  return await page.evaluate(() =>
+    Array.prototype.map.call(document.querySelectorAll('a'), (a) => {
+      // There's a weird serialization boundary here so we encode the links as a structure
```

Comment:
For context, Puppeteer talks to Chrome via [Chrome DevTools Protocol](https://chromedevtools.github.io/devtools-protocol/), which is what you encountered here.

Reply:
sweet, will update the comment to be more descriptive

Reply:
0008fef2fb6d4a067f3685472ec53bd441cec3b7